### PR TITLE
Added history command and updated DG

### PR DESCRIPTION
**Pull request by EdmundTangg**
#68

### CODE REVIEW

**Comment by leongxingyu** [comment]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -71,6 +70,29 @@ The **API** of this component is specified in [`Ui.java`](https://github.com/AY2
...
+
+**Methods**
+* `NewCommand`: Constructor for creating a new instance of the NewCommand class.
+* `execute`: Method to add the new product to the product list.
+* `createProduct`: Method to create a new product with a unique product ID.
+
+The following sequence diagram shows how an add operation works:
+<img src="images/AddCommandSequence.png" alt=""/>
+
+
```
Comment (NgYaoDong) [inline docs/DeveloperGuide.md]: this should be under Implementation section, not Architecture

File: src/main/java/seedu/stockpal/data/ProductList.java [hunk 2]
```diff
@@ -78,20 +78,23 @@ public void updateProduct(int productIndex, Name newName, Quantity newQuantity
      * @param productIndex Product PID to update
      * @param amountToIncrease Quantity of product to decrease
      */
```
Comment (Kobot7) [inline src/main/java/seedu/stockpal/data/ProductList.java:81]: Would be good to include return value in JavaDoc

**Comment by wjunjie01** [comment]
LGTM

**Comment by cheeseong2001** [comment]
Lgtm

**Pull request merged**